cloud_modules/azure_rm.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create and remove an Azure virtual machine
+  hosts: localhost
+  connection: local
+  gather_facts: False
+
+  tasks:
+    - name: Create VM
+      azure_rm_virtualmachine:
+        admin_username: "{{ admin_username | default('tower-qe') }}"
+        admin_password: "{{ admin_password | default('YouSh@llNotPas5') }}"
```

Comment:
this is a public repo maybe we should not put this info here or generate something random